### DISCUSSION TITLE
how to pass options to ffmpeg?

### DISCUSSION THREAD

**Discussion by loddar66** [crunchy-labs/crunchy-cli#387] [Q&A] [answered]
Hi, when it comes to merging the streams into one mkv container, ffmpeg already does a decent job. But I did not find out how to finetune the settings to better fit my needs when it e.g. comes to bitrate etc. 
The documentation says: 
_You can specify specific built-in presets with the `--ffmpeg-preset` flag to convert videos to a specific coding while downloading. Multiple predefined presets how videos should be encoded (h264, h265, av1, ...) are available, you can see them with crunchy-cli download --help. If you need more specific ffmpeg customizations you could either convert the output file manually or use ffmpeg output arguments as value for this flag._
What's the exact syntax of the 'output arguments' to be passe to ffmpeg? I would like to use something like "`-c:v h264_amf -preset fast -b:v 3000k`" instead of the built-in `--ffmpeg-preset` option.

Thanks in advance!

**Comment by bytedream** [comment, marked as answer]
Its `--ffmpeg-preset="-c:v h264_amf -preset fast -b:v 3000k"`

So a full command could be:
```shell
$ crunchy-cli download --ffmpeg-preset="-c:v h264_amf -preset fast -b:v 3000k" https://www.crunchyroll.com/watch/GRDKJZ81Y/alone-and-lonesome
```

**Reply by loddar66** [reply]
thanks, I'll try it...

**Reply by loddar66** [reply]
works as intended- thank you for your support!
